.github/scripts/release-against-charts.sh

```diff
@@ -36,6 +36,12 @@ for i in fleet fleet-crd fleet-agent; do
     yq --inplace ".${i} += [\"${NEW_CHART_VERSION}+up${NEW_FLEET_VERSION}\"]" release.yaml
 done
 
+# Adapt Gitjob version in generated patch
+if grep -q '^-  version: ' ./packages/fleet/fleet/generated-changes/patch/Chart.yaml.patch; then
```

Comment:
This check is needed to skip the check for fleet >0.10.x where gitjob is included?